src/main/java/uk/gov/ons/census/casesvc/service/QuestionnaireLinkedService.java

```diff
@@ -98,4 +115,37 @@ private void logUnlinkedCaseEventAgainstOriginalCaseIfQidIsNowLinkedToNewCase(
           messageTimestamp);
     }
   }
+
+  private boolean checkRequiredFieldsForIndividualHI(
+      String questionnaireId, String caseType, String individualCaseId) {
+    if (isIndividualQuestionnaireType(questionnaireId)
+        && caseType.equals("HH")
+        && individualCaseId != null) {
+      return true;
```

Comment:
These can be made **return (isIndividualQuestionnaireType(questionnaireId)
        && caseType.equals("HH")
        && individualCaseId != null)**